### ISSUE TITLE
jamesleedemo/dockerapp not found

### ISSUE THREAD

**Issue by rbpaservices** [jleetutorial/dockerapp#10]
Following the UDEMY course The complete Docker Course Section 5 Lecture 30 refers to a fork of this app that I cannot find.  Please advise
![image](https://cloud.githubusercontent.com/assets/17012484/23107353/e24abfec-f6c9-11e6-981c-666378f5fdea.png)

**Comment by dimitriacosta** [comment]
Actually you should fork this repo into your GitHub account so you'll be able to make changes to the code into your own repository

**Comment by jleetutorial** [comment]
@rbpaservices : Yes, you would need to fork this repo into your GitHub account so that have permissions to make changes. Just click the folk button on the jleetutorial/dockerapp

**Comment by rbpaservices** [comment]
I did that and I got the permissions error you spoke of when I tried to
push the changes back.  Although I cloned it from my repo, and I removed it
and cloned it again to be sure(I also removed the forked dockerapp and
forked it again too) , it seems to be trying to push changes to
jamesleetutorial/dockerapp.  There must be something else that I missed
